bootloader/src/uart_receive_multiple_timeout.rs

```diff
@@ -209,7 +209,7 @@ impl<'a, A: hil::time::Alarm<'a>> hil::uart::ReceiveClient for UartReceiveMultip
                     //
                     // Receive up to half of the buffer at a time so there is
                     // room if the host sends us more than we expect.
-                    self.uart.receive_buffer(buffer, buffer.len() / 2);
+                    self.uart.receive_buffer(buffer, 1);
```

Comment:
It would be good to know how much this slows down transfers on the nano33 -- we could probably make this value configurable

Reply:
I agree with having this value variable, I do think it will slow down the nano 33 ble.

Reply:
I added a variable value